influxql/ast.go

```diff
@@ -372,7 +372,7 @@ func (s *CreateUserStatement) String() string {
 	_, _ = buf.WriteString("CREATE USER ")
 	_, _ = buf.WriteString(s.Name)
 	_, _ = buf.WriteString(" WITH PASSWORD ")
-	_, _ = buf.WriteString(s.Password)
```

Comment:
@dgnorton I lie earlier when I said this is a preemptive change. This statement can be logged if authorization fails. Not the worst thing in the world, but not ideal.

```
[query] 2015/07/22 14:46:08 unauthorized request | user: "test" | query: "CREATE USER test2 WITH PASSWORD [REDACTED]" | database "mydb"
```